### PR TITLE
READY TO MERGE: traefik ingress-per-domain java integ test

### PR DESCRIPTION
**Pull request by maggiehe00**
1. Add 2 new attributes into domainonpvwlst.yaml and domainonpvwdt.yaml  "loadBalancer: TRAEFIK" and "ingressperDomain: true";
2. Class Domain: new Boolean variable ingressPerDomain to match "ingressPerDomain" value in the 
   domain.yaml; 
   2 keys are added into ldMap: ingressPerDomain and clusterName
3. Class LoadBalancer: TRAEFIk ingressPerDomain will be created if lbMap.get("ingressPerDomain") is true
4. Add env var INGRESSPERDOMAIN with defult value is true. If you want to test creating LB by kubectl yaml for multiple domains, set it to false
Jenkins passed at job: http://wls-jenkins.us.oracle.com/view/weblogic-operator/job/weblogic-kubernetes-operator-javatest/1079/
after merging with Vanaja's latest change "get hostname from nslookup for curl" and the above item 4 change.

### CODE REVIEW

**Comment by vanajamukkara** [comment]
@maggiehe00 Can you see this comment?

File: integration-tests/README.md [hunk 1]
```diff
@@ -204,6 +204,7 @@ The tests accepts optional env var overrides:
 | RESULT_ROOT | The root directory to use for the tests temporary files. See "Directory Configuration and Structure" for                  defaults and a detailed description of test directories. |
 | PV_ROOT    |  The root directory on the kubernetes cluster used for persistent volumes. See "Directory Configuration and Structure" for defaults and a detailed description of test directories. |
 | QUICKTEST  | When set to "true", limits testing to a subset of the tests. |
+| YAMLSTYLEINGRESS  | When set to "true", LB's ingress will be created by kubectl yaml file. |
```
[the remark this answers is not in the document]
Comment (maggiehe00) [reply integration-tests/README.md]: @vanajamukkara I'm going to address this

File: integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java [hunk 2]
```diff
@@ -978,6 +995,13 @@ private void initialize(Map<String, Object> inputDomainMap) throws Exception {
     clusterName = (String) domainMap.get("clusterName");
     clusterType = (String) domainMap.get("clusterType");
     serverStartPolicy = (String) domainMap.get("serverStartPolicy");
+    /*TODO remove mhe
```
[the remark this answers is not in the document]
Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java]: @vanajamukkara This is removed.

File: integration-tests/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (maggiehe00) [reply integration-tests/README.md]: @vanajamukkara I changed env var to INGRESSPERDOMAIN with default value: true. I also tested: when it is set to false hostrouting style ingress is created; when it is not set or is set to true ingress-per-domain style ingress is created.

**Comment by maggiehe00** [comment]
@vanajamukkara The latest Jenkins passed at :http://wls-jenkins.us.oracle.com/view/weblogic-operator/job/weblogic-kubernetes-operator-javatest/1079/

**Pull request merged**